### PR TITLE
daniel/tsk-3566: add tenable integration

### PR DESCRIPTION
**Pull request by blink-daniel**

### CODE REVIEW

File: openapi/mask.go [hunk 1]
```diff
@@ -68,14 +69,11 @@ func (m *Plugin) GetParameter(actionName string, paramName string) *MaskedAction
...
+func (m *Plugin) GetAction(actionName string) (action *MaskedAction, ok bool) {
 	originalActionName := m.ReplaceActionAlias(actionName)
+	action, ok = m.Actions[originalActionName]
 
-	if action, ok := m.Actions[originalActionName]; ok {
-		return action
-	}
-
-	return nil
+	return
```
Comment (ozhey) [inline openapi/mask.go:76]: `return m.Actions[originalActionName]`

File: plugins/tenable/mask.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      required: true
+collection:
+  name: tenable
+  icon_uri: https://app.dev.blinkops.com/assets/images/Icons/Companies/Company/XL/Tenable.svg
+  active: true
+is_connection_optional: false
+actions:
+  agents-list:
+  bulk-add-agents:
+  scanners-list:
```
Comment (ozhey) [inline plugins/tenable/mask.yaml:28]: Looks like some of the parameter display names are really not clear like f, w, wf, tf and similar
Let's use the mask to fix it

File: openapi/mask.go (hunk 1 above)
Comment (blink-daniel) [reply openapi/mask.go:74]: this won't compile (return m.Actions[originalActionName] will return only the map value without the ok)

Comment (ozhey) [reply openapi/mask.go:74]: Oh, ok cool

**Review by ozhey — APPROVED** [review]

**Pull request merged**